apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/PublisherClient.cs

```diff
@@ -220,10 +220,9 @@ internal void Validate()
         /// <see cref="PublisherServiceApiClient"/>s.</param>
         /// <param name="settings">Optional. <see cref="Settings"/> for creating a <see cref="PublisherClient"/>.</param>
         /// <returns>A <see cref="PublisherClient"/> instance associated with the specified <see cref="TopicName"/>.</returns>
+        [Obsolete("TODO - instructions for PublisherClientBuilder, or maybe don't make it obsolete")]
```

Comment:
I would even make ClientCreationSettings obsolete.

Reply:
Maybe introduce the builder in the next release, and only make things obsolete a bit later? It's a shame there isn't a "suggestion-only" obsolete to allow us to ramp up even more gradually.